libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1434,6 +1434,11 @@
         "comment": "Label shown on dropdown button that changes form of the attribute used in insight. For example 'Display as: Shortened name'",
         "limit": 0
     },
+    "attributesDropdown.empty.item.tooltip": {
+        "value": "By selecting this item, missing (null) and empty values will be included.",
```

Comment:
Should this differ between bear that has no support for nulls and tiger that supports them?